### PR TITLE
tests(db): test `compareScore()` and `getAreVotedBySessionId()`

### PR DESCRIPTION
**Pull request by niklasmtj**
This should help a little with #267

### CODE REVIEW

File: utils/db_test.ts [hunk 1]
```diff
@@ -205,3 +208,49 @@ Deno.test("[db] createComment() + getCommentsByItem()", async () => {
   await assertRejects(async () => await createComment(comment2));
   assertArrayIncludes(await getCommentsByItem(itemId), [comment1, comment2]);
 });
+
+Deno.test("[db] compareScore()", () => {
```
Comment (iuioiua) [inline utils/db_test.ts:212]: Another way we could do this is to create an items array in the wrong order by score, then sort it via `compareScore()` and check the new order. This doesn't improve anything in a technical sense but does convey the intent and behaviour of the use of `compareScore()` to the reader. WDYT?

**Review by iuioiua — CHANGES_REQUESTED** [review]

File: utils/db_test.ts (hunk 1 above)
Comment (niklasmtj) [reply utils/db_test.ts:212]: Yes, that sounds like a good idea! Will implement it towards the end of the week :)

Comment (iuioiua) [reply utils/db_test.ts:212]: No worries!

File: utils/db_test.ts [hunk 2]
```diff
@@ -205,3 +208,49 @@ Deno.test("[db] createComment() + getCommentsByItem()", async () => {
...
+    title: crypto.randomUUID(),
+    url: `http://${crypto.randomUUID()}.com`,
+    ...newItemProps(),
+    score: 2,
+  };
+
+  assertEquals(compareScore(item1, item2), 1);
+});
+
+Deno.test("[db] getAreVotedBySessionId()", async () => {
```
[the remark this answers is not in the document]
Comment (niklasmtj) [reply utils/db_test.ts:261]: Check the commit [19e3e33](https://github.com/denoland/saaskit/pull/300/commits/19e3e3335d333d682033257a0a1ec9f9fb67ae95) if this is what you thought :)

File: utils/db_test.ts (hunk 1 above)
Comment (niklasmtj) [reply utils/db_test.ts:242]: Updated it in [153f423](https://github.com/denoland/saaskit/pull/300/commits/153f42317f81284fe78d366bc4df643e9ed3afce). This is way better than before. I also added a third item to not just switch the items from left to right.

**Comment by niklasmtj** [comment]
Interesting that the tests are failing on [Linux](https://github.com/denoland/saaskit/actions/runs/5432086805/jobs/9878895545?pr=300). Is there something I can do about it?

File: utils/db_test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (niklasmtj) [reply utils/db_test.ts:271]: Or wait you'll probably mean something like 

```
 assertArrayIncludes(await getAreVotedBySessionId([item], null), []);
```

Do you?

[the remark this answers is not in the document]
Comment (iuioiua) [reply utils/db_test.ts:271]: More like:
```ts
assertEquals(await getAreVotedBySessionId([item], undefined), []);
```

**Comment by iuioiua** [comment]
The failure on Linux is due to an endpoint error when downloading imports. Unfortunately, it happens occasionally.

File: utils/db_test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (niklasmtj) [reply utils/db_test.ts:271]: Got you. That example came to my mind just before I opened this PR 😄

[the remark this answers is not in the document]
Comment (niklasmtj) [reply utils/db_test.ts:271]: What do you prefer `assertEquals` or `assertArrayIncludes` in this case?

[the remark this answers is not in the document]
Comment (iuioiua) [reply utils/db_test.ts:271]: `assertEquals` will do the trick.

**Review by iuioiua — APPROVED** [review]
LGTM! Nicely done! Thank you.

**Pull request merged**